src/github.com/getlantern/peerscanner/web.go

```diff
@@ -149,12 +149,29 @@ func clientIpFor(req *http.Request, name string) string {
 	ip := strings.TrimSpace(ips[0])
 	// TODO: need a more robust way to determine when a non-fallback host looks
 	// like a fallback.
-	if !isFallback(name) && strings.HasPrefix(ip, "128.199") {
```

Comment:
When I merged the changes from devel, I pulled in this very narrow definition of what is a fallback.  I've expanded it to include ips that are in use in the CloudFlare Amsterdam data center.